src/models/HybridSystem.jl

```diff
@@ -0,0 +1,172 @@
+"""
+Representation of a Hybrid System that collects renewable generation, load, thermal generation
+and storage
+"""
+mutable struct HybridSystem{
+    T <: ThermalGen,
+    L <: ElectricLoad,
+    S <: Storage,
+    R <: RenewableGen,
+} <: StaticInjection
+    name::String
+    available::Bool
+    status::Bool
+    bus::Bus
+    active_power::Float64
+    reactive_power::Float64
+    thermal_unit::T
+    electric_load::L
+    storage::S
+    renewable_unit::R
+    # PCC Data
+    "Thermal limited MVA Power Output of the unit. <= Capacity"
+    pcc_impedance::ComplexF64
```

Comment:
I'm torn on the use of `pcc`. It's a fairly standard abbreviation, but it's still an abbreviation.  Also, is the `pcc_` even neede, could it be `impedance`.

Reply:
I am fine with the change in name, just thought this would make it more clear

Reply:
I switched to interconnection